integration/build.gradle.kts

```diff
@@ -40,20 +40,12 @@ tasks {
 
     run.configure {
         environment("TF_CPP_MIN_LOG_LEVEL" to "1") // turn off TensorFlow print out
-        // @Niels Doucet
-        // Just a heads-up: gradle support warned me about systemProperties System.getProperties(). It's really
-        // dangerous to just copy over all system properties to a task invocation. You should really be specific about
-        // the properties you'd like to expose inside the task, or you might get very strange issues.
-        systemProperties = System.getProperties().toMap() as Map<String, Any>
```

Comment:
What's the right way to pass system properties in command line when run gradle?

```
gradle run -Dai.djl.offline=true
```

There are many DJL specific system properties, it's hard for use the list them

Reply:
If they are all namespaced like that example, you can copy over the properties that start with `ai.djl.offline` instead of blanket copying over all System properties. That should be much safer overall, and also avoid copying properties which change frequently and 'break' the incremental builds.